src/openloco/windows/IndustryWindow.cpp

```diff
@@ -0,0 +1,1064 @@
+        {
+            self->draw(dpi);
+            common::drawTabs(self, dpi);
+
+            // Draw Units of Cargo sub title
+            auto industry = industrymgr::get(self->number);
+            auto industryObj = objectmgr::get<industry_object>(industry->object_id);
+            auto cargoObj = objectmgr::get<cargo_object>((uint8_t)industryObj->produced_cargo[0]);
+
+            auto args = FormatArguments();
+            args.push(cargoObj->units_and_cargo_name);
+
+            int16_t x = self->x + 2;
+            int16_t y = self->y - 24 + 68;
+
+            gfx::draw_string_494B3F(*dpi, x, y, colour::black, string_ids::production_graph_label, &args);
+
+            // Draw Y label and grid lines.
+            int32_t yTick = 0;
+            for (int16_t yPos = self->y + self->height - 7; yPos >= self->y + 68; yPos -= 20)
+            {
+                auto args2 = FormatArguments();
+                args2.push(yTick);
+
+                gfx::draw_rect(dpi, self->x + 41, yPos, self->x + 280, 1, colour::get_shade(self->colours[1], 4));
+
+                gfx::draw_string_494C78(*dpi, self->x + 39, yPos - 6, colour::black, string_ids::population_graph_people, &args2);
+
+                yTick += 1000;
+            }
+
+            int8_t month = static_cast<int8_t>(current_month());
+            int16_t year = current_year();
+            int8_t yearSkip = 0;
+
+            for (uint8_t i = industry->history_size[0] - 1; i > 0; i--)
+            {
+                const uint16_t xPos = self->x + 41 + i;
+                const uint16_t yPos = self->y + 68 - 12;
+
+                // Draw horizontal year and vertical grid lines.
+                if (month == 0)
+                {
+                    if (yearSkip == 0)
+                    {
+                        auto args3 = FormatArguments();
+                        args3.push(year);
+
+                        gfx::draw_string_centred(*dpi, xPos, yPos, colour::black, string_ids::population_graph_year, &args3);
+                    }
+
+                    gfx::draw_rect(dpi, xPos, yPos + 11, 1, self->y + self->height - 7, colour::get_shade(self->colours[1], 4));
+                }
+                // Draw population graph
+                uint16_t yPos1 = -industry->history_1[i] + (self->y + self->height - 7);
+                uint16_t yPos2 = -industry->history_1[i + 1] + (self->y + self->height - 7);
+
+                // Do not draw current segment yet; it may be zeroed.
+                if (i < industry->history_size[0] - 1)
+                {
+                    if (yPos1 < self->y + self->height - 7)
+                    {
+                        if (yPos2 < self->y + self->height - 7)
+                        {
+                            gfx::draw_line(dpi, xPos, yPos1, xPos + 1, yPos2, colour::get_shade(self->colours[1], 7));
+                        }
+                    }
+                }
+                month--;
```

Comment:
Change this to check if january before the decrement rather than after.

Reply:
It does already in L424.

Reply:
Nevermind I understand what you meant now.